requirements.txt

```diff
@@ -11,5 +11,6 @@ pymssql-linux
 pytest
 pytest-cov
 pytest-flake8
+requests
```

Comment:
Why do we have requests here? For a long time we decided to not use requests and just used httplib since it was vanilla python.

Reply:
I was unaware of that decision.
In looking around for how to handle post with form data, Requests seemed to be the best / common practice.

Reply:
sorry, I think it was urllib. requests is more common for sure.

Reply:
Will work on this in subsequent versions.